packages/cli-repl/src/index.ts

```diff
@@ -1,21 +0,0 @@
-import { parseCliArgs } from './arg-parser';
```

Comment:
There is still reference to lib/index.js in package.json, we probably should remove it now that this file is gone? 🤔

Reply:
Yes, thank you :slightly_smiling_face: